terraform/modules/public-load-balancer/variables.tf

```
@@ -1,3 +1,9 @@
+variable "allowlist_cidrs" {
+  type        = list(string)
+  default     = ["0.0.0.0/0"]
+  description = "Allowlist for HTTPS access (not necessarily all paths)."
```

Can we make it more specific? e.g. would this be accurate?
```suggestion
  description = "Allowlist for HTTPS access (but doesn't allow access to paths in restricted_path_patterns)."
```